README.md

```diff
@@ -1,7 +1,25 @@
-# Sphinx Symbiflow Theme
+# SymbiFlow's Sphinx Theme (Based on Material Design).
+
+This repository contains the [Sphinx](https://www.sphinx-doc.org/en/master/)
+ theme used
+[by SymbiFlow](https://symbiflow.github.io)
+on all it's generated documentation
+([including the theme itself!](https://github.com/SymbiFlow/sphinx_symbiflow_theme)).

```

Comment:
* Wanna link to the generated documentation (https://sphinx-symbiflow-theme.readthedocs.io/en/latest/) instead?
* including the theme's itself

Reply:
Change the link.

Change the wording to `include the theme's own!`.